packages/extension-chakra-storefront/src/components/field/index.jsx

```diff
@@ -111,8 +111,9 @@ const Field = ({
 
                         {type === 'select' && (
                             <>
-                                <NativeSelect.Root ref={ref} value={value} {..._inputProps}>
+                                <NativeSelect.Root value={value} {..._inputProps}>
                                     <NativeSelect.Field
```

Comment:
This is needed for the React Hook Form to properly access the field for validation